mayastor/src/bdev/nexus/mod.rs

```diff
@@ -59,3 +59,17 @@ pub fn nexus_instance_new(name: String, size: u64, children: Vec<String>) {
     let list = instances();
     list.push(Nexus::new(&name, size, None, Some(&children)));
 }
+
+/// called during shutdown so that the bdevs for each child is destroyed first
+pub async fn nexus_destroy_all() {
+    debug!("destroying all nexuses...");
```

Comment:
should these be info? will only happen on shutdown and might be useful to see them if for some reason we crash while shutting down

Reply:
I thought I was trying to be consistent with some of the other log entries but at least "NVMe-oF target shutdown completed" and "Unloading Nexus CAS Module" are at info level. I'll change these.